fix(AI/ContextChat): update docs (#12529)
nextcloud/documentation: admin_manual/ai/app_context_chat.rst

   occ config:app:set context_chat indexing_max_jobs_count --value=3 --type=integer

* ``request_timeout`` integer (default: 3000)

>    Request timeout in seconds for all requests made to the backend.

Reviewer: This could be a bit more explicit.
```suggestion
   Request timeout in seconds for all requests made to the Context chat backend (the external app in AppAPI).
```

>    ``TIMEOUT_SERVER`` in the docker socket proxy should be set to a value higher than this.

Reviewer: ```suggestion
   If a docker socket proxy is used, the ``TIMEOUT_SERVER`` environment variable should be set to a value higher than ``request_timeout``.
```

.. code-block::

   occ config:app:set context_chat request_timeout --value=3 --type=integer

Configuration Options (Backend)
-------------------------------

Refer to `the Configuration head <https://github.com/nextcloud/context_chat_backend?tab=readme-ov-file#configuration>`_ in the backend's readme.

Logs
----

Logs for the ``context_chat`` PHP app can be found in the Nextcloud log file, which is usually located in the Nextcloud data directory. The log file is named ``nextcloud.log``.

| For the backend, warning and error logs can be found in the docker container logs, and the complete logs can be found in ``logs/`` directory in the persistent storage of the docker container.
| That will be ``/nc_app_context_chat_backend/logs/`` in the docker container.
| See `the Logs head <https://github.com/nextcloud/context_chat_backend?tab=readme-ov-file#logs>`_ in the backend's readme for more information.

Possibility of Data Leak
------------------------

> | It is possible that some users who have been denied access to certain files/folders still have access to the content of those files/folders through the Context Chat app. We're working on a solution for this.

Reviewer: ```suggestion
| It is possible that some users who had access to certain files/folders (and have later have been denied this access) still have access to the content of those files/folders through the Context Chat app. We're working on a solution for this.
```

Author: cool
but why the leading `|`?

Author: ah nevermind
